### DISCUSSION TITLE
[Enhancement] [image] 我想在函数中调用预览打开和预览关闭的接口，现在应该怎么做

### DISCUSSION THREAD

**Discussion by pyeden** [element-plus/element-plus#17377] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `image`

## Description

我想在函数中调用预览打开和预览关闭的接口，现在应该怎么做

### Additional comments

我想在函数中调用预览打开和预览关闭的接口，现在应该怎么做

**Comment by warmthsea** [comment]
#### reference: 

![code](https://github.com/element-plus/element-plus/assets/45450994/c685e4fa-334c-4c19-aa0f-6fa91aec05ee)
